S3TransferUtilitySample/src/com/amazonaws/demo/s3transferutility/DownloadActivity.java

```diff
@@ -336,6 +344,7 @@ public void onClick(View arg0) {
         });
 
         updateButtonAvailability();
+        requestPermission();
```

Comment:
change function name to `requestWriteExternalStoragePermission()`

Reply:
done!